vignettes/frictionless.Rmd

```diff
@@ -72,48 +70,51 @@ str(package, list.len = 3)
 #>   .. ..$ profile  : chr "tabular-data-resource"
 #>   .. .. [list output truncated]
 #>   [list output truncated]
+#>  - attr(*, "class")= chr [1:2] "datapackage" "list"
 ```
 
 The most important aspect of a Data Package are its **Data Resources**, which describe and point to the data. You can list all included resources with `resources()`:
 
+
 ```r
 resources(package)
 #> [1] "reference-data" "gps"            "acceleration"
 ```
 
 This Data Package has 3 resources. Let's read the data from the `gps` resource into a data frame:
 
+
 ```r
-package <- read_package("https://zenodo.org/record/5879096/files/datapackage.json")
 gps <- read_resource(package, "gps")
 gps
 #> # A tibble: 73,047 × 21
-#>     `event-id` visible timestamp           `location-long` `location-lat`
-#>          <dbl> <lgl>   <dttm>                        <dbl>          <dbl>
-#>  1 14256075762 TRUE    2018-05-25 16:11:37            4.25           51.3
-#>  2 14256075763 TRUE    2018-05-25 16:16:41            4.25           51.3
-#>  3 14256075764 TRUE    2018-05-25 16:21:29            4.25           51.3
-#>  4 14256075765 TRUE    2018-05-25 16:26:28            4.25           51.3
-#>  5 14256075766 TRUE    2018-05-25 16:31:21            4.25           51.3
-#>  6 14256075767 TRUE    2018-05-25 16:36:09            4.25           51.3
-#>  7 14256075768 TRUE    2018-05-25 16:40:57            4.25           51.3
-#>  8 14256075769 TRUE    2018-05-25 16:45:55            4.25           51.3
-#>  9 14256075770 TRUE    2018-05-25 16:50:49            4.25           51.3
-#> 10 14256075771 TRUE    2018-05-25 16:55:36            4.25           51.3
+#>     `event-id` visible timestamp           `location-long` `location-lat` bar:barometric-pressur…¹
```

Comment:
If I knit `vignettes/frictionless.Rmd.orig` via `compile.R` I get a different output. For example, `bar:barometric-pressure` instead of `bar:barometric-pressur…¹`  in this line.
I noticed that everytime I change the window size and/or the font size I get a new result. Is this a Windows specific behavior? Can you check on your MacOS, please? It's a quite annoying behavior.

Reply:
Note: this `option` will reset itself if you close the session, so it doesn't affect the developer that much.